src/app/dataset-view/additional-components/metadata-component/components/form-components/array-keys-field/array-keys-field.component.ts

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable @typescript-eslint/unbound-method */
```

Comment:
A general problem - can we avoid defining the pragmas for entire file, and instead, use them only very close to the usage point?

Reply:
Yes, of course.